hail/python/hailtop/utils/utils.py

```diff
@@ -233,6 +236,8 @@ def is_transient_error(e):
         return True
     if isinstance(e, ConnectionResetError):
         return True
+    if isinstance(e, google.auth.exceptions.TransportError):
+        return True
```

Comment:
`TransportError` seems to be raised in response to a more fundamental error, so it seems we should
```python3
return is_transient_error(e.__cause__)
```